### PR TITLE
Use 20.08 runtime

### PR DESCRIPTION
**Pull request by tintou**

### CODE REVIEW

**Comment by hadess** [comment]
Can you please split this up into 2? This also begs the question of why the flatpak-external-data-checker bot didn't run. This will need fixing first I think.

**Comment by hadess** [comment]
> This also begs the question of why the flatpak-external-data-checker bot didn't run. This will need fixing first I think.

Looks like we should read http://www.sweethome3d.com/history.jsp instead, and use the latest version from that. The front page isn't always updated when new minor versions come out.

**Comment by tintou** [comment]
@hadess Alright, fixed your requests :slightly_smiling_face:

**Comment by hadess** [comment]
> @hadess Alright, fixed your requests slightly_smiling_face

If that last commit generated with flatpak-external-data-checker?

**Comment by tintou** [comment]
@hadess no it is not but `flatpak-external-data-checker` returns no issue with it

File: com.sweethome3d.Sweethome3d.json [hunk 1]
```diff
@@ -64,4 +64,4 @@
             ]
         }
     ]
-}
```
Comment (hadess) [inline com.sweethome3d.Sweethome3d.json:67]: Can you remove this change please?

(the easiest way being to run:
`flatpak-external-data-checker --commit-only com.sweethome3d.Sweethome3d.json` :)

**Comment by hadess** [comment]
> @hadess no it is not but `flatpak-external-data-checker` returns no issue with it

There's really no better way to show that the changes work than by using them. I had to test the changes locally to make sure that the checker metadata worked as expected (which it does).

**Comment by hadess** [comment]
https://github.com/flathub/com.sweethome3d.Sweethome3d/pull/25 contains the data checker fix, we should soon see an automated MR for the version update. Can you rebase the runtime change when that's done?

**Comment by hadess** [comment]
Updating to 6.5.2 was done through the automated PR here: https://github.com/flathub/com.sweethome3d.Sweethome3d/pull/26

**Comment by tintou** [comment]
I've update the pull request

**Pull request merged**